x-pack/plugins/uptime/server/rest_api/uptime_route_wrapper.ts

```diff
@@ -26,34 +26,22 @@ export const uptimeRouteWrapper: UMKibanaRouteWrapper = (uptimeRoute) => ({
       esClient: esClient.asCurrentUser,
     });
 
-    try {
-      const res = await uptimeRoute.handler({
-        uptimeEsClient,
-        savedObjectsClient,
-        context,
-        request,
-        response,
-      });
-
-      if (res instanceof KibanaResponse) {
-        return res;
-      }
-
-      return response.ok({
-        body: {
-          ...res,
-        },
-      });
-    } catch (e) {
-      // please don't remove this, this will be really helpful during debugging
-      /* eslint-disable-next-line no-console */
-      console.error(e);
+    const res = await uptimeRoute.handler({
```

Comment:
We've removed this `try...catch` and `console.error` call presumably because an error will result in a detailed stack trace/message logged anyway, and otherwise we return `res` or a create a new response.

Is that correct?

Reply:
Exactly, we log the error as-is. So the stack trace should have the same result than the `console.error` there.